cms/tests/frontend/integration/nonadmin.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+var helpers = require('djangocms-casper-helpers');
+var globals = helpers.settings;
+var casperjs = require('casper');
+var xPath = casperjs.selectXPath;
+var cms = helpers(casperjs);
+
+casper.test.setUp(function(done) {
+    casper
+        .start()
+        .then(cms.login())
+        .then(cms.addPage({ title: 'Home page' }))
+        .then(
+            cms.addPlugin({
+                type: 'TextPlugin',
```

Comment:
please use a plugin from the cms tests instead.
like style, etc..

Reply:
thisisfine.jpg